src/GBACart.cpp

```diff
@@ -691,6 +691,42 @@ void CartRAMExpansion::ROMWrite(u32 addr, u16 val)
     }
 }
 
+CartRumblePak::CartRumblePak() : CartCommon()
+{
+}
+
+CartRumblePak::~CartRumblePak()
+{
+}
+
+void CartRumblePak::Reset()
+{
+    RumbleState = 0;
+}
+
+void CartRumblePak::DoSavestate(Savestate* file)
+{
+    CartCommon::DoSavestate(file);
+    file->Var16(&RumbleState);
+}
+
+u16 CartRumblePak::ROMRead(u32 addr) const
+{
+    // TODO: Verify this
+    return 0xFFFD;

```

Comment:
Are you sure this shouldn't be `addr & 1 ? addr : (addr | 0x0040) & 0xFFFD`, as per the previous PR?

Reply:
> Are you sure this shouldn't be `addr & 1 ? addr : (addr | 0x0040) & 0xFFFD`, as per the previous PR?

The solution used in the previous PR was a cheap hack used before proper open bus behavior was implemented.